system-addon/content-src/components/Search/_Search.scss

```diff
@@ -84,3 +85,8 @@
     transform: translateY(2px);
   }
 }
+
+// The search dropdown isn't themed (yet), so we need to keep dark text.
+#searchSuggestionTable {  // sass-lint:disable-line no-ids
```

Comment:
There's already a `.contentSearchSuggestionTable` just a few lines higher that we can reuse.

```diff
@@ -79,7 +80,25 @@
   }
+}
 
+@at-root {
   // Adjust the style of the contentSearchUI-generated table
-  .contentSearchSuggestionTable { // sass-lint:disable-line class-name-format
+  // sass-lint:disable-block class-name-format
+  .contentSearchSuggestionTable {
+    background-color: var(--newtab-searchbox-background-color);
     border: 0;
-    transform: translateY(2px);
+    transform: translateY(3px);
+
+    .contentSearchHeader {
+      background-color: var(--newtab-background-color);
+      color: var(--newtab-text-secondary-color);
+    }
+
+    .contentSearchHeader,
+    .contentSearchSuggestionsList {
+      border-color: var(--newtab-border-primary-color);
+    }
+
+    .contentSearchSearchWithHeaderSearchText {
+      color: var(--newtab-text-primary-color);
+    }
   }
```
![screen shot 2018-03-27 at 9 30 15 pm](https://user-images.githubusercontent.com/438537/38008901-3c1f0e88-3206-11e8-9be6-b13a69486425.png)
![screen shot 2018-03-27 at 9 30 24 pm](https://user-images.githubusercontent.com/438537/38008905-3e2e5d0a-3206-11e8-911d-0dc4e1788503.png)

Reply:
Oh, and I wonder if we should name the `--var`s as `--newtab-search-…` (no "box") ?

Reply:
(I used `@at-root` mostly to create a new block for the sass disable.)

Reply:
awesome, thanks!. and --newtab-search- sounds good. fixing